beacon-chain/types/state.go

```diff
@@ -112,6 +126,9 @@ func (a *ActiveState) NewPendingAttestation(record *pb.AttestationRecord) {
 
 // LatestPendingAttestation returns the latest pending attestaton fields.
 func (a *ActiveState) LatestPendingAttestation() *pb.AttestationRecord {
```

Comment:
This method is also untested. Please consider adding a test for this method and the new lines added